client/pokemon/pokemon.ctrl.js

```diff
@@ -69,11 +71,12 @@ module.exports = class Pokemon {
       `${this.data.formName ? '-' + this.data.formName : ''}`;
   }
   parseAllProps () {
+    this.uploadGen = this.data.gen;
     this.parseBoxViewProps();
 
+    this.tidFull = this.data.tidFull;
```

Comment:
Hmmm possibly. But we can decide that later on, it's not related to the PR. It's probably worth doing it **after** release as well, just so we don't have lots of problems all at once. Check the code works well in prod, and then look at migrating.